### DISCUSSION TITLE
What it a recommended way of integration with Microsoft Search (SharePoint/Teams)

### DISCUSSION THREAD

**Discussion by xperiandri** [requarks/wiki#4854] [Help / Questions] [answered]
As I see from [docs page](https://docs.microsoft.com/en-us/microsoftsearch/connectors-overview#what-data-sources-can-be-connected-to-microsoft-search) there are different connectors ([catalog](https://www.microsoft.com/microsoft-search/connectors/?category=&query=)) that allow plugging external items to a search index that can be consumed from Office 365 search boxes (like Office.com, Teams, SharePoint, etc.)
Is any of them can consume Wiki.js?

**Comment by NGPixel** [comment, marked as answer]
Not at the moment, no.
